polyfill/lib/yearmonth.mjs

```diff
@@ -52,7 +52,7 @@ export class YearMonth {
     const disambiguation = ES.ToArithmeticTemporalDisambiguation(options);
     const duration = ES.ToLimitedTemporalDuration(temporalDurationLike);
     let { year, month } = this;
```

Comment:
Is this statement correct: There's no arithmetic in MonthDay, which is why you're touching YearMonth but not MonthDay.